### PR TITLE
AC Automated Test - LRAC-8836 | master

### PR DESCRIPTION
**Pull request by rafaellajordao**
https://issues.liferay.com/browse/LRAC-8836

### CODE REVIEW

**Comment by rafaellajordao** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase [hunk 1]
```diff
@@ -115,12 +115,21 @@ definition {
...
-		// TODO LRAC-8836 CancelCreatingDxpDataSource
+			ACNavigation.goToSettings();
+
+			ACSettings.goToDataSources();
 
+			ACSettings.clickGenericAddOrConfirmationButton();
+
+			Click(locator1 = "AnalyticsCloudConnection#LIFERAY_DXP_SOURCE");
+
+			ACNavigation.clickCancelButton();
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase:131]: Hi @rafaellajordao, I think you can check in the data source list if a new data source has not been created

Comment (rafaellajordao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase:131]: Good idea! @ViniLopes20 fixed!!

**Comment by rafaellajordao** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase [hunk 2]
```diff
@@ -115,12 +115,27 @@ definition {
...
+		}
+
+		task ("Assert the data source list if a new data source has not been created") {
+			AssertElementNotPresent(
+				itemName = "Liferay",
+				locator1 = "ACSettings#ITEM_ON_LIST");
+		}
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase:138]: hi @rafaellajordao , I think when we click on "Cancel" the user is not redirected to the list of data sources, I think you would need to navigate back one more step, so I think this assertion where it is now will not work. :🤔  Furthermore, I don't think validate for the "Liferay" string is safe, if we had another Data Source with this name, this assertion will fail.

One suggestion here would be getting the number of existing data sources on the list before trying to create a new one, e.g.: `Showing 1 to 9 of 9 entries.` Then, after canceling you can just compare if that number is still the same.

What do you think?

**Comment by alexcandidos** [comment]
Hi @rafaellajordao , could you please take a look at the suggestions? Thanks!

**Comment by rafaellajordao** [comment]
new pr:
https://github.com/liferay-qa/liferay-portal/pull/867

**Pull request closed without merging**